index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>My portfolio</title>
+	<link rel="shortcut icon" type="image/jpg" href="images/favicon.png"/>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="normalize.css" />
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+	<link rel="stylesheet" href="portfoliostyle.css" />
+</head>
+<body>
+	<nav>
+		<h1>William Thornhill</h1>
+		
+		<div class="nav-buttons">
+			<button type="button">About me</button>
+		
+			<button type="button">Portfolio</button>
+		
+			<button type="button">Contacts</button>
+		</div>
+	</nav>
+	
+	<main>
+	<div class="about-projects">
```

Comment:
Potentially don't need this div that contains about and projects - if you've done it so you can style them similarly then you could just use both classes in your css selector? :)

Reply:
This was done so I could Flex them in a particular way, I will try fixing it

Reply:
Ahhh I get you- when they go into 2 columns at a smaller screen size! That looks cool. My apologies!

Reply:
It's fine, thanks for the feedback